artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/JournalImpl.java

```diff
@@ -3565,6 +3580,8 @@ protected void moveNextFile(final boolean scheduleReclaim, boolean blockOnClose)
 
       if (scheduleReclaim) {
          scheduleReclaim();
+      } else {
+         logger.trace("JournalImpl::moveNextFile scheduleReclaim is false ( {} ), not calling scheduleReclaim", scheduleReclaim);
```

Comment:
Seems odd to have a text statement it is false, and then also a placeholder printing the same value as well (also the arg for that will be autoboxing the boolean...not necessarily an issue if it isnt especially hot or typically does scheduleReclaim)

Reply:
When I wrote this, I was concerned about the value changing between the check and the if value. so I was just reprinting just in case.. but I guess it can go away.

Reply:
The value looks to be coming from a final method parameter and so cant change.